File: src/main/java/org/example/kalkulationsprogramm/domain/Dokument.java
package org.example.kalkulationsprogramm.domain;

import java.time.LocalDate;

public interface Dokument {
    Long getId();
    String getOriginalDateiname();
    String getGespeicherterDateiname();
    String getDateityp();
    Long getDateigroesse();
    LocalDate getUploadDatum();
    LocalDate getEmailVersandDatum();
    DokumentGruppe getDokumentGruppe();
}

File: src/main/kotlin/org/example/kalkulationsprogramm/domain/Dokument.kt
package org.example.kalkulationsprogramm.domain

import java.time.LocalDate

interface Dokument {
    val id: Long?
    val originalDateiname: String?
    val gespeicherterDateiname: String?
    val dateityp: String?
    val dateigroesse: Long?
    val uploadDatum: LocalDate?
    val emailVersandDatum: LocalDate?
    val dokumentGruppe: DokumentGruppe?
}
